CRM/Utils/System/Backdrop.php

```diff
@@ -1212,24 +1212,16 @@ public function theme(&$content, $print = FALSE, $maintenance = FALSE) {
       \CRM_Core_Error::deprecationMessage('CRM_Utils_System::theme called with $maintenance = TRUE - please use renderMaintenanceMessage instead');
     }
 
-    $ret = FALSE;
-
     if (!$print) {
       if ($maintenance) {
         print $this->renderMaintanceMessage($content);
```

Comment:
Umm.
Also the spelling of the actual function override in this file.

Reply:
Sorry also down on line 1230, where the override is.

Reply:
Wow ok. I'm surprised this hasn't been more symptomatic! But increases my resolve to strip most of this out...